third_party/ienn/src/ie_compilation.cpp

```diff
@@ -368,7 +436,26 @@ int32_t Compilation::AddConvolution(const Operation& operation) {
     const uint32_t output_index = operation.outputs[0];
     // Since cldnn doesn't support mixed layout
     // We reshape the bias to 4D NCHW aligned with conv_node
-    AddConstant(bias_index, false, {1, params.bias_length, 1, 1});
+    if (index_op_map_.find(bias_index) == index_op_map_.end()) {
+      // Setup constants
+      if (model_->values.find(bias_index) != model_->values.end()) {
+        if (model_->operands[bias_index].dimensions.size() == 1) {
+          result =
+              AddConstant(bias_index, false, {1, params.bias_length, 1, 1});
+          ;
+        } else {
+          result = AddConstant(bias_index, true);
```

Comment:
What's the rank of the bias? is it special for quantized model?

Reply:
The bias rank could be 1 or 4 in models. The else branch is just for handling the 4-rank and we need to transpose it from nhwc to nchw.